Intersect.Client/Networking/PacketHandler.cs

```diff
@@ -1014,29 +1014,18 @@ public void HandlePacket(IPacketSender packetSender, EntityAttackPacket packet)
                 return;
             }
 
-            if (!packet.IsBlocking)
+            var isSelf = en == Globals.Me;
+            if (isSelf)
             {
-                //attack handler
-                if (attackTimer > -1 && en != Globals.Me)
-                {
-                    en.AttackTimer = Timing.Global.Ticks / TimeSpan.TicksPerMillisecond + attackTimer;
-                    en.AttackTime = attackTimer;
-                }
+                en.IsBlocking = packet.IsBlocking;
```

Comment:
I just realized that my suggestion is not actually compatible with your logic, this is my bad.

I think this actually needs to be (in order to match the functionality you were adding):

```
if (isSelf && packet.IsBlocking)
{
    en.IsBlocking = attackTimer > -1;
}
```